packages/arb-token-bridge-ui/src/components/TransactionHistory/PendingDepositWarning.tsx

```diff
@@ -21,7 +18,7 @@ export function PendingDepositWarning() {
           <InformationCircleIcon className="mt-[2px] h-4 w-4 shrink-0 stroke-orange-dark" />
           <p>
             Deposit might fail if the gas fee provided was too low. Stay on this
-            page until the {layer} transaction succeeds.
+            page until the transaction succeeds.
```

Comment:
We use a generic copy without network names because it's not reliable. User could have different networks selected to an actual tx chains in tx history.

Or they could have multiple erc-20 deposits on different chains.